test/configs/modularity.json

```diff
@@ -1,5 +1,8 @@
 {
   "name": "modularity",
+  "expect_modularity": {
+    "modules": ["nodejs:18"]
+  },
```

Comment:
Maybe instead explicitly enable the module in the blueprint and base your check script on that? :thinking:

Reply:
I think this testcase is from before the enabled modules in blueprints and was based on the package-name-is-module-spec bits. I'll add a second testcase that uses the enabled modules.

Reply:
I've added a second test config for this.